app/routes.py

```diff
@@ -1,2 +1,274 @@
-from flask import Blueprint
+import requests
+from dotenv import load_dotenv
+import os
+import datetime
+from sqlalchemy import desc
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
 
+load_dotenv()
+
+
+@tasks_bp.route("/", methods=["GET", "POST"])
+def handle_tasks():
+    if request.method == "GET":
+        sort_query = request.args.get("sort")
+        if sort_query == "asc":
+            tasks = Task.query.order_by(Task.title)
+        elif sort_query == "desc":
+            tasks = Task.query.order_by(desc(Task.title))
+        else:
+            tasks = Task.query.all()
+
+        tasks_response = []
+        for task in tasks:
+            if not task.completed_at:
+                is_complete = False
+            else:
+                is_complete = True
+            tasks_response.append({
+                "id": task.task_id,
+                "title": task.title,
+                "description": task.description,
+                "is_complete": is_complete
+            })
```

Comment:
There are many places in your routes where you build a dictionary like this (or very similar). Consider making a helper method, either here in the routes file (e.g. `def task_to_dict(task)`) or as an instance method of the `Task` model class (e.g. `def to_dict(self)`).